### PR TITLE
DX-2242 Migrate AMD docs 

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

All PRs on this repo that change any API sources of truth (markdown files, OpenAPI specs) require a changelog added to the `site/src/pages/changelog.md` file. If this PR does not require changelog updates, you need to add the `no-changelog` tag to this PR before opening it.

Please confirm that you have either updated `site/src/pages/changelog.md` or added the `no-changelog` tag.

### CODE REVIEW

File: site/docs/voice/webhooks/machineDetectionComplete.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+| applicationId          | The id of the application associated with the call. |
+| to                     | The phone number that received the call, in E.164 format (e.g. +15555555555). |
+| from                   | The provided identifier of the caller: can be a phone number in E.164 format (e.g. +15555555555) or one of `Private`, `Restricted`, `Unavailable`, or `Anonymous`. |
+| direction              | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes. |
+| callId                 | The call id associated with the event. |
+| callUrl                | The URL of the call associated with the event. |
+| startTime              | Time the call was started, in ISO 8601 format. |
+| answerTime             | Time the call was answered, in ISO 8601 format. |
+| machineDetectionResult | The result of the Machine Detection Analysis. Possible values are `answering-machine`, `human`, `silence`, `timeout`, or `error`. |
+| tag                    | (optional) The `tag`  specified on call creation. If no `tag` was specified or it was previously cleared, this field will not be present. |
```
Comment (CatexB) [inline site/docs/voice/webhooks/machineDetectionComplete.mdx:47]:
```suggestion
| tag                    | (optional) The `tag` specified on call creation. If no `tag` was specified or it was previously cleared, this field will not be present. |
```

File: site/docs/voice/machineDetectionGuide.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+id: machineDetectionGuide
+title: Machine Detection
+slug: /voice/guides/machineDetection
+description: How to using Bandwidth's Voice API Answering Machine Detection Feature
```
Comment (CatexB) [inline site/docs/voice/machineDetectionGuide.mdx:5]:
```suggestion
description: How to use Bandwidth's Voice API Answering Machine Detection Feature
```

File: site/docs/voice/machineDetectionGuide.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  - answering
+  - machine
+  - detection
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+This guide will show you how to use machine detection, using the `machineDetection` parameter in a [call creation request](/apis/voice#operation/createCall).
+
+You can use either a sync or an async mode for this operation.
```
Comment (tmoney-bw) [inline site/docs/voice/machineDetectionGuide.mdx:18]:
```suggestion
You can choose to use either synchronous (sync) or asynchronous (async) mode for this operation.
```

"a sync" and "an async" seems potentially confusing on a screen reader or just to someone scanning the documentation quickly

File: site/docs/voice/machineDetectionGuide.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+title: Machine Detection
+slug: /voice/guides/machineDetection
+description: How to using Bandwidth's Voice API Answering Machine Detection Feature
+keywords:
+  - bandwidth
+  - voice
+  - answering
+  - machine
+  - detection
+hide_title: false
```
Comment (n-miles) [inline site/docs/voice/machineDetectionGuide.mdx:12]: Maybe `AMD` should be a keyword?

**Review by hamermike — APPROVED** [review]

**Review by hamermike — APPROVED** [review]

**Review by ckoegel — APPROVED** [review]

**Pull request merged**